redwood-treehouse-host/src/androidUnitTest/kotlin/app/cash/redwood/treehouse/TreehouseLayoutTest.kt

```diff
@@ -112,7 +115,9 @@ class TreehouseLayoutTest {
           bottom = insets.bottom.toDp(),
         )
       }
-      assertThat(awaitItem().safeAreaInsets).isEqualTo(expectedInsets)
+      val value2 = awaitItem()
+      assertThat(value2.safeAreaInsets).isEqualTo(Margin.Zero)
+      assertThat(value2.viewInsets).isEqualTo(expectedInsets)
```

Comment:
Change log item for this behavior change? Probably want to give a heads-up to internal users as well, if they still exist, that is.